src/server/pachyderm_test.go

```diff
@@ -4315,7 +4315,7 @@ func testGetLogs(t *testing.T, enableStats bool) {
 		}
 
 		// Get logs from pipeline, using pipeline (tailing the last two log lines)
-		iter = c.GetLogs(pipelineName, "", nil, "", false, false, 2)
+		iter = c.GetLogs(pipelineName, "", nil, "", true, false, 2)
```

Comment:
What was the reasoning behind this change? Is it related to this PR or just a separate thing that was fixed?

Reply:
This is related to this PR but also due to how unreliable `GetLogs` is, which we've discussed separately.  Mainly that PPS filters the log statements after applying the limit and it is very likely to get an empty result when there are valid log statements that match the filter.  In the new worker, log statements differ enough that filtering for master logs is more likely to return something.

Reply:
Makes sense.